youtube_dl/postprocessor/embedthumbnail.py

```diff
@@ -52,7 +52,7 @@ def run(self, info):
             if not check_executable('AtomicParsley', ['-v']):
                 raise EmbedThumbnailPPError('AtomicParsley was not found. Please install.')
 
-            cmd = ['AtomicParsley', filename, '--artwork', temp_thumbnail, '-o', temp_filename]
+            cmd = ['AtomicParsley', filename, '--artwork', thumbnail_filename, '-o', temp_filename]
```

Comment:
Filenames should be `encodeFilename`'ed, otherwise non-ASCII will make subprocess fail on python2 @ Windows.